_includes/breadcrumbs.html

```diff
@@ -1,11 +1,11 @@
 <div id="breadcrumbs" class="pb-3">
     {% assign crumbs = page.url | remove:'/index.html' | split: '/' %}
-    <a href="/">Home</a>
+    <a href="/">{% t generic.home %}</a>
     {% for crumb in crumbs offset: 1 %}
       {% if forloop.last %}
-        / {{ page.title }}
+        / {% capture title %}{% t page.title %}{% endcapture %}{{ title }}
```

Comment:
That's a bit weird. You capture something just to print it directly afterwards. Wouldn't it be enough to just use `{% t page.title %}`?

Reply:
Didn't work before (I think document titles containing `{{ page.slug }}` caused this to fail), but now the document titles are settled with a full proper reference, `{% t page.title %}` seems to work fine indeed.